requirements-dev.txt

```diff
@@ -1,8 +1,12 @@
 #
-# This file is autogenerated by pip-compile
-# To update, run:
+# NOTE: for the time being, requirements-dev.txt
+# cannot be stably compiled with pip-compile
+# due to an inconsistent pinned version of the
+# six package.
 #
-#    pip-compile requirements-dev.in
+# First upgrade all outdated dependencies in
+# requirements.txt and then compile dev deps
+# again
```

Comment:
Would be nice if this problem with six could be fixed first and then this note here wouldn't be necessary and dependency management could be used normally.

Reply:
Ah, this was based on my wrong usage. Removed, fixed.